e2e-commons/down.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+cd $(dirname $0)
+
+if [ $CI ]; then exit $rc; fi
+
+ps | grep node | grep -v grep | awk '{print "kill " $1}' | /bin/bash
```

Comment:
could you clarify what `node` process is expected here?

Reply:
`node ./scripts/blocks.js &` that can be found in `up.sh`

Reply:
thanks!

I would simplify it a bit:
```
ps | grep node | grep -v grep | awk '{print $1}' | xargs kill
```